jobs/build/merge_ocp/merge_ocp.sh

```diff
@@ -29,6 +29,6 @@ echo 'pkg/assets/bindata.go merge=ours' >> .gitattributes
 echo 'pkg/assets/java/bindata.go merge=ours' >> .gitattributes
 git merge -m "Merge remote-tracking branch ${UPSTREAM_SOURCE_BRANCH}" "upstream/${UPSTREAM_SOURCE_BRANCH}"
```

Comment:
This is already a no-op when there's nothing new. I was confused because of what the build job does -- creates and extra commit even if it's empty. So anyway :+1:

Reply:
Yep... should've clarified there. It's the build job that's adding that :)